x/evm/migrations/migrate_eip_1559_params_test.go

```diff
@@ -0,0 +1,27 @@
+package migrations_test
+
+import (
+	"testing"
+
+	testkeeper "github.com/sei-protocol/sei-chain/testutil/keeper"
+	"github.com/sei-protocol/sei-chain/x/evm/migrations"
+	"github.com/sei-protocol/sei-chain/x/evm/types"
+	"github.com/stretchr/testify/require"
+	tmtypes "github.com/tendermint/tendermint/proto/tendermint/types"
+)
+
+func TestMigrateEip1559Params(t *testing.T) {
+	k := testkeeper.EVMTestApp.EvmKeeper
+	ctx := testkeeper.EVMTestApp.NewContext(false, tmtypes.Header{})
+
+	// Perform the migration
+	err := migrations.MigrateEip1559Params(ctx, &k)
```

Comment:
can we modify some param before the migration and see if the modification is reverted afterwards?

Reply:
the comment is wrong actually, should not be set back to defaults. Did set a param before the migration to check it works correctly.